Tests/FrameworkTests/EmptyFileTests.swift

```diff
@@ -0,0 +1,75 @@
+import SwiftLintFramework
+import XCTest
+
+private protocol BooleanConstant {
```

Comment:
I'm not really a big fan of this, but looks like there is no other way due to the rules being statically initialized.